addons/udes_stock/data/company.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="base.main_company" model="res.company">
+      <field name="name">UDES</field>
+    </record>
+
+    <!-- Details for partner --> 
+    <record id="base.main_partner" model="res.partner">
+      <field name="email">noreply@unipart.com</field>
```

Comment:
Note: Not sure about using this email... just make a note here for my own benefit.

Reply:
Closing, will look more into it and handle separately if needed.